src/IO/InterfileHeader.cxx

```diff
@@ -620,6 +624,36 @@ InterfilePDFSHeader::InterfilePDFSHeader()
   add_key("Reference energy (in keV)",
           &reference_energy);
 
+  // new keys for block geometry
+  scanner_orientation = "X";
+  add_key("Scanner orientation (X or Y)",
+          KeyArgument::ASCII, &scanner_orientation);
+
+  scanner_geometry = "Cylindrical";
+  add_key("Scanner geometry (BlocksOnCylindrical/Cylindrical/Generic)",
+          KeyArgument::ASCII, &scanner_geometry);
+
+  axial_distance_between_crystals_in_cm = -0.1;
```

Comment:
these defaults are still negative. To me this doesn't make sense. They should be zero, as that's what our default cylindrical scanner has.